im.dino.Dino.yml

```diff
@@ -0,0 +1,57 @@
+---
+app-id: im.dino.Dino
+runtime: org.gnome.Platform
+runtime-version: "42"
+sdk: org.gnome.Sdk
+command: dino
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11" # for whatever reason breaks with fallback-x11 on GNOME on Wayland
+  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--share=network"
+  - "--device=all" # required for audio/video calling
+  - "--talk-name=org.gtk.vfs.*"
+  - "--filesystem=xdg-run/gvfsd"
+modules:
+  - name: gspell
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gspell/1.8/gspell-1.8.3.tar.xz
+        sha256: 5ae514dd0216be069176accf6d0049d6a01cfa6a50df4bc06be85f7080b62de8
+  - name: libsignal-protocol-c
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_C_FLAGS=-fPIC
+    sources:
+      - type: git
+        path: plugins/signal-protocol/libsignal-protocol-c
+        url: https://github.com/WhisperSystems/libsignal-protocol-c.git
+        tag: v2.3.3
+  - name: qrencode
+    buildsystem: cmake-ninja
+    cleanup:
+      - /bin
+    config-opts:
+      - -DCMAKE_C_FLAGS=-fPIC
+    sources:
+      - type: archive
+        url: https://fukuchi.org/works/qrencode/qrencode-4.1.1.tar.gz
+        sha512: 209bb656ae3f391b03c7b3ceb03e34f7320b0105babf48b619e7a299528b8828449e0e7696f0b5db0d99170a81709d0518e34835229a748701e7df784e58a9ce
+  - name: dino
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/dino/dino.git
+        tag: v0.3.0
+        commit: 9838d5679470d1add098accaeae8eaf0ee3c58ee
+  - name: dino-misc
+    buildsystem: simple
+    build-commands:
+    - install -D im.dino.Dino.appdata.xml /app/share/appdata/im.dino.Dino.appdata.xml
+    sources:
+    - type: file
+      path: im.dino.Dino.appdata.xml
```

Comment:
```suggestion
  - name: dino
    buildsystem: cmake-ninja
    builddir: true
    post-install:
      - install -D ../im.dino.Dino.appdata.xml /app/share/appdata/im.dino.Dino.appdata.xml
    sources:
      - type: git
        url: https://github.com/dino/dino.git
        tag: v0.3.0
        commit: 9838d5679470d1add098accaeae8eaf0ee3c58ee
      - type: file
        path: im.dino.Dino.appdata.xml
```

Reply:
Actually changing the suggestion. An AppStream file is already installed by upstream so you don't need one here.